src/patterns/PaymentForm.vue

```diff
@@ -4,7 +4,12 @@
       type="hidden"
       name="stripeToken"
       value="">
-    <div :class="b('form-control').toString()">
+    <div
```

Comment:
This is a lot of logic to have inline. Any chance you could move this into a function? Just a stylistic preference for readability and maintainability.